app/views/searches/_form.html.erb

```diff
@@ -1,4 +1,21 @@
 <form class="search_form form-inline">
+  <%= react_component("search/Filters", {
+    allForms: [
+      {
+        id: "1",
+        displayName: "Form A"
+      },
+      {
+        id: "2",
+        displayName: "Form B"
+      },
+      {
+        id: "3",
+        displayName: "Form C"
+      }
+    ],
+    selectedFormIds: ["2"]
+  }) %>
```

Comment:
This is only an example; eventually this state needs to be provided dynamically. I assume we'll wait to merge until that's ready too?